### PR TITLE
feat: CAIP-29 for ERC1155 assets

### PR DESCRIPTION
**Pull request by shrugs**
An asset reference and token id specification for blockchain references to ERC1155 tokens on an eip155 chain. Based on the CAIP-22 specification.

### CODE REVIEW

File: CAIPs/CAIP-XX.md
Comment (ligi) [inline CAIPs/CAIP-XX.md:2]:
```suggestion
caip: 29
```

**Review by ligi — CHANGES_REQUESTED** [review]
Looks really good to me - thanks for the PR
Only change to CAIP-29 (filename and header)
and add to the README
Thanks for the PR!

**Comment by shrugs** [comment]
done! thanks for the quick review.

**Comment by ligi** [comment]
You are most welcome! Would still give some other reviewers some time to look over it before merging though. Hope @antoineherzog @pedrouid and @oed can approve or give comments here as they have most domain knowledge in this area.
Hope you still have a bit of time to wait for them. Otherwise I also feel quite confident already merging - don't think they will have significant objections in this particular case.

**Review by oed — APPROVED** [review]
lgtm

**Review by antoineherzog — APPROVED** [review]
LGTM!

**Review by ligi — APPROVED** [review]

**Comment by shrugs** [comment]
@ligi are there any extra pending approvals?

**Comment by ligi** [comment]
I hoped @pedrouid would also approve - but I think 3 is good enough

**Pull request merged**
